stack.yaml

```diff
@@ -19,6 +19,7 @@ packages:
 # (e.g., acme-missiles-0.3)
 extra-deps:
   - avro-0.2.0.0
+  - aeson-lens-0.5.0.0
```

Comment:
This should come from package.yaml as I understand.

Reply:
I think it's because it isn't published to `stackage`